packs/st2/actions/workflows/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,37 @@
+---
+  chain:
+    -
+      name: "check_for_permission_to_send_anonymous_data"
+      ref: "st2.kv.get"
+      params:
+        key: "st2::collect_anonymous_data"
+      on-success: "get_server_uuid"
```

Comment:
I had this, but then recalled a bug where the action shows as successfully executed if the final task in a chain was successful, even if we took the negative path

What I wanted to avoid was having someone freak out when they saw a task labeled "call home" and saw a green big arrow. rather, red meant "nope, didn't do it", and when they see why, it's failed on the "check for permission" task.

Also, where do I output that data? this is an async task happening outside of user involvement.